docs/docs/how-tos/domain-registry.md

```diff
@@ -0,0 +1,12 @@
+---
+id: domain-registry
+title: How to set Nebari's DNS Domain Registry
+---
+
+# Nebari domain registry setup
+
+This domain will be where your application will be exposed.
+
+Currently, Nebari only supports CloudFlare for automatic DNS registration. If an alternate DNS provider is desired, change the `--dns-provider` flag from **cloudflare** to **none** on the Nebari deploy command. The deployment then will be paused when it asks for an IP address (or CNAME, if using AWS) and prompt to register the desired URL.
```

Comment:
```suggestion
Currently, Nebari only supports CloudFlare for automatic DNS registration. If an alternate DNS provider is desired, change the `--dns-provider` flag from **cloudflare** to **none** on the Nebari deploy command. The deployment process will then pause when it asks for an IP address (or CNAME, if using AWS) and prompt you to provide the desired URL.
```

I *think* this is an improvement... without having actually done a deployment, I can't be sure if I understand what's happening here, but "register" seems like the wrong word for a user pasting an IP/CNAME into the CLI.

Reply:
oh! I just populated these files with part of what's currently present in our docs -- That whole section needs to be written!

Reply:
for ref. https://docs.qhub.dev/en/stable/source/installation/setup.html?highlight=cloudflare#domain-registry

Reply:
No, I think it's ok - it looks like it just got split to a new paragraph.